analyzers/src/SonarAnalyzer.CSharp/Rules/EqualityOnModulus.cs

```diff
@@ -72,5 +80,12 @@ private static bool IsUnsigned(ExpressionSyntax expression, SemanticModel semant
             return type.IsAny(KnownType.UnsignedIntegers)
```

Comment:
Lets create a field containing both the UnsignedIntegers and the UintPtr and using that with .IsAny().